drivers/vmwarevcloudair/vcloudair.go

```diff
@@ -407,66 +411,17 @@ func (d *Driver) Create() error {
 		return err
 	}
 
-	if d.Provision {
-		dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
-
-		log.Infof("Installing Docker...")
-
-		cmd, err = d.GetSSHCommand(dockerInstall)
-		if err != nil {
-			return err
-		}
-
-		if err = cmd.Run(); err != nil {
-			return err
-		}
+	dockerInstall := "curl -sSL https://get.docker.com/ | sudo sh"
```

Comment:
Guys, not a problem, I just picked that from the page, but if there's a better way to do the install, I'm all for that.

And for the Note, copy and paste never lies :wink:

![https://www.dropbox.com/s/4jtk58zd5hfsgk2/Screenshot%202015-01-13%2011.02.38.png?dl=0](https://www.dropbox.com/s/4jtk58zd5hfsgk2/Screenshot%202015-01-13%2011.02.38.png?dl=1)